input/pagecontent/conformance-patients.md

```diff
@@ -1,14 +1,14 @@
-To facilitate conformance testing, testing software must be able to determine which patients are "mCODE Patients" -- those in scope for mCODE. In general, all patients with confirmed cancer diagnoses SHOULD be covered by mCODE. In FHIR terms, these are patients who have a Condition where `Condition.code` is a member of the value set [PrimaryOrUncertainBehaviorCancerDisorderVS] and `Condition.verificationStatus` is confirmed.
+To facilitate conformance testing, testing software must be able to determine which patients are in-scope -- Cancer patients who are in the scope of mCODE. In general, all patients with confirmed cancer diagnoses SHOULD be in-scope. In FHIR terms, these are patients who have a Condition where `Condition.code` is a member of the value set [PrimaryOrUncertainBehaviorCancerDisorderVS] and `Condition.verificationStatus` is confirmed.
 
-However, due to technical, organizational, or legal reasons, mCODE Data Senders MAY exclude some cancer patients from mCODE. In that case, the mCODE Data Sender MUST define a Group resource to identify ALL mCODE patients in their system. This Group resource MUST set `Group.code` to `mcode-patient` with code system `http://hl7.org/fhir/us/mcode/CodeSystem/mcode-resource-identifier-cs`. Data Senders that do not exclude any cancer patients from mCODE MAY still populate a `mcode-patient` Group resource.
+However, due to technical, organizational, or legal reasons, mCODE Data Senders MAY exclude some cancer patients from mCODE. In that case, the mCODE Data Sender MUST define a Group resource to identify ALL in-scope patients in their system. This Group resource MUST set `Group.code` to `mcode-patient` with code system `http://hl7.org/fhir/us/mcode/CodeSystem/mcode-resource-identifier-cs`. Data Senders that do not exclude any cancer patients from mCODE MAY still populate a `mcode-patient` Group resource.
```

Comment:
Note that if we change `#mcode-patient` above, we need to update this too.

Reply:
I suggest keeping the current code.

Reply:
Keep it.